src/Workspaces/Core/Portable/CodeActions/CodeActionPriority.cs

```diff
@@ -2,25 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.CodeAnalysis.Diagnostics;
+
 namespace Microsoft.CodeAnalysis.CodeActions
 {
+#pragma warning disable CA1200 // Avoid using cref tags with a prefix
+    /// <summary>
+    /// Internal priority used to bluntly place items in a light bulb in strict orderings.  Priorities take
+    /// the highest precedence when ordering items so that we can ensure very important items get top prominance,
+    /// and low priority items do not.
+    /// </summary>
+    /// <remarks>
+    /// If <see cref="CodeActionPriority.High"/> is used, the feature that specifies that value should 
+    /// implement and return true for <see cref="IBuiltInAnalyzer.IsHighPriority"/>,
+    /// <see cref="T:Microsoft.CodeAnalysis.CodeFixes.CodeFixProvider.IsHighPriority"/> and
```

Comment:
I think `T:` is unneccary and the suppression can be removed?

Reply:
this is the workspaces layer talking about types in teh features layer.